MM-28882 Add config to Ignore Guests on AD/LDAP Sync (#4149)
mattermost/docs: source/deployment/sso-saml-ldapsync.rst

To configure SAML synchronization with AD/LDAP:

1. Go to **System Console > Authentication > SAML 2.0** (or **System Console > SAML** in versions prior to 5.12) and set **Enable Synchronizing SAML Accounts With AD/LDAP** to ``true``.

> 2. Go to  **System Console > Authentication > SAML 2.0** and set **Ignore Guest Users when Synchronizing with AD/LDAP** to ``true`` or ``false``. 

Reviewer: The original step 2 is still valid. It should now be step 3.

Author: Very good catch, Scott. Thank you for the oversight!

Author: Should be resolved with my latest commit. Also updated the ignore step, given its default value.

3. Set the rest of the AD/LDAP settings based on `configuration settings documentation <http://docs.mattermost.com/administration/config-settings.html#ad-ldap>`__ to connect Mattermost with your AD/LDAP server.

 - If you don't want to enable AD/LDAP sign-in, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and keep **Enable sign-in with AD/LDAP** as ``false``.

> 4. To specify how often Mattermost synchronizes SAML user accounts with AD/LDAP, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and set **Synchronization Interval**. The default setting is 60 minutes. If you want to synchronize immediately after disabling an account, click the **AD/LDAP Synchronize Now** button.

Reviewer 2: ```suggestion
4. To specify how often Mattermost synchronizes SAML user accounts with AD/LDAP, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and set **Synchronization Interval**. The default setting is 60 minutes. If you want to synchronize immediately after disabling an account, click **AD/LDAP Synchronize Now**.
```

> 5. To test Mattermost can successfully connect to your AD/LDAP server, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and click the **AD/LDAP Test** button.

Reviewer 2: ```suggestion
5. To confirm that Mattermost can successfully connect to your AD/LDAP server, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and select **AD/LDAP Test**.
```
